docs/conf.py

```diff
@@ -224,7 +227,12 @@
 #latex_preamble = ''
 
 # Documents to append as an appendix to all manuals.
-#latex_appendices = []
+latex_appendices = [
+    'support',
+    'resources',
+    'changes',
+    'authors',
+]
```

Comment:
The "all manuals" part in the docstring means all *latex* manuals, since it's possible to generate multiple distinct manuals from the same Sphinx project. (E.g. one manual for the Python stdlib, one for the CPython API, one for the language spec. Or a user's guide and a developer's guide.) We don't do that, though.